Userland/Libraries/LibWeb/CSS/CSSStyleSheet.h

```diff
@@ -50,8 +50,8 @@ class CSSStyleSheet final
 
     void set_style_sheet_list(Badge<StyleSheetList>, StyleSheetList*);
 
-    Optional<StringView> default_namespace() const;
-    Optional<StringView> namespace_uri(StringView namespace_prefix) const;
+    Optional<String> default_namespace() const;
+    Optional<String> namespace_uri(StringView namespace_prefix) const;
```

Comment:
Let's make it an `Optional<FlyString>` right away, since that's what `StyleComputer` compares against this anyway. :)

Reply:
Ah right, that would be even nicer :^)